src/logging.c

```diff
@@ -56,8 +56,31 @@ bool g_rcutils_logging_severities_map_valid = false;
 
 int g_rcutils_logging_default_logger_level = 0;
 
+bool g_force_stdout_line_buffered = false;
+bool g_stdout_flush_failure_reported = false;
+
+
 rcutils_ret_t rcutils_logging_initialize(void)
 {
+  const char * line_buffered;
+  const char * ret_str = rcutils_get_env("RCUTILS_CONSOLE_STDOUT_LINE_BUFFERED", &line_buffered);
+
+  if (NULL == ret_str) {
+    if (strcmp(line_buffered, "1")) {
+      g_force_stdout_line_buffered = true;
+    } else {
+      if (!strcmp(line_buffered, "0")) {
+        fprintf(stderr,
+          "Warning: unexpected value %s specified for RCUTILS_CONSOLE_STDOUT_LINE_BUFFERED."
+          "Default value 0 will be used. Valid values are 1 or 0.\n",
+          line_buffered);
+      }
+    }
+  } else {
+    if (NULL != ret_str) {
+      fprintf(stderr, "Error getting env var: %s\n", ret_str);
```

Comment:
Please add the name of the environment variable here so that the reader has a better idea where it is failing.